renderer/utils/html_audio_plugin.js

```diff
@@ -0,0 +1,45 @@
+var HTMLAudioPlayer = {
+    _audios : {},
+    _audioLimit : 25,

```

Comment:
Increase the audio limit. For the single letter audios(keyboard button audio) may have more than 25 audios.
Increase to max 50. Make sure cleaning of audios, when it is unused.